purchase_advance_payment/static/description/index.html

```diff
@@ -449,4 +448,4 @@ <h2><a class="toc-backref" href="#toc-entry-7">Maintainers</a></h2>
 </div>
 </div>
 </body>
-</html>
+</html>
```

Comment:
Looks like you are removing newline. Should fix your editor to not mess up file endings:)

Reply:
This was pre-commit fix I think.

Reply:
Hm, if its done by pre-commit, it looks like its doing something wrong. Usually it should do the opposite, cause now github is throwing "warning":)

Reply:
Yep, looks like it was editor not pre-commit after all...